application/apps/rustcore/ts-bindings/spec/session.benchmark.spec.ts

```diff
@@ -0,0 +1,306 @@
+                                        stream
+                                            .observe(
+                                                new Factory.File()
+                                                    .type(Factory.FileType.PcapNG)
+                                                    .file(`${home_dir}/${test.file}`)
+                                                    .asDlt({
+                                                        filter_config: undefined,
+                                                        fibex_file_paths: [],
+                                                        with_storage_header: false,
+                                                        tz: undefined,
+                                                    })
+                                                    .get()
+                                                    .sterilized(),
+                                            )
+                                            .catch(finish.bind(null, session, done));
+                                        break;
+                            case 'Stream - startup measurement':
+                                const tmpobj1 = createSampleFile(
+                                    5000,
+                                    logger,
+                                    (i: number) => `some line data: ${i}\n`
+                                );
+
+                                const startupSession = await Session.create();
+                                startupSession.debug(true, testName);
+
+                                const startupStream = startupSession.getStream();
+                                if (startupStream instanceof Error) throw startupStream;
+
+                                const startupEvents = session.getEvents();
+                                if (startupEvents instanceof Error) throw startupEvents;
+                                startupStream.observe(
+                                    new Factory.Stream()
+                                        .asText()
+                                        .process({
+                                            command: `less ${tmpobj1.name}`,
+                                            cwd: process.cwd(),
+                                            envs: process.env as { [key: string]: string },
+                                        })
+                                        .get()
+                                        .sterilized()
+                                );
+                                const startupResults = measurement();
+                                const startupReport = performanceReport(testName, startupResults.ms, test.expectation_ms, `${home_dir}/${test.file}`);
+                                finish(startupSession, done, startupReport ? undefined : new Error(`${testName} is fail`));
+                                break;
```

Comment:
you are leaving a test without closing `session` which was created above at line 42